src/corelibs/U2View/src/ov_sequence/Overview.cpp

```diff
@@ -48,7 +48,7 @@ const QString Overview::ANNOTATION_GRAPH_STATE = "sequenceViewSettings/annotatio
 Overview::Overview(ADVSingleSequenceWidget *p, ADVSequenceObjectContext *ctx)
     : GSequenceLineView(p, ctx),
       seqWidget(p) {
-    renderArea = new OverviewRenderArea(this);
+    renderArea = overviewRenderArea = new OverviewRenderArea(this);
```

Comment:
Please, do not do several assignments on one line. It is all about style, so I don't have good arguments, but I don't like it.

Reply:
Ok, will fix in the next commit inside this PR.
BTW I would prefer to have both these fields *const ptrs but can't find a good way to write it - it can only be done in the constructor initializers.

Reply:
Done